plugins/catalog-node/src/api/processor.ts

```diff
@@ -100,6 +100,14 @@ export type CatalogProcessor = {
     emit: CatalogProcessorEmit,
     cache: CatalogProcessorCache,
   ): Promise<Entity>;
+
+  /**
+   * Returns a processor priority, which is used to determine the order in which
+   * processors are run. The default priority is 20, and lower value means
+   * that the processor runs earlier.
+   * @returns A number representing the priority of the processor.
+   */
+  getPriority?(): number;
```

Comment:
I was wondering for a bit whether we could make this a config-only thing but I agree that it makes sense to have as code as well to make it easy for people to just add individual new things without being forced to configure as well

Reply:
Yep, I think the "default" should be possible to be defined by the processor creator, but still keep the option to go around it with config.